docs/state.md

```diff
@@ -186,3 +186,32 @@ if err != nil {
 gcsClient, err := storage.New(httpClient)
 
 ```
+
+## Vault (vault://)
+
+The goal of the vault store is to be a safe storage for the kops keys and secrets store. It will not work to use this as a kops registry/config store. Among other things, etcd-manager is not able to read VFS control file from vault. Vault can also not be used as backend for etcd backups.
+
+The vault store uses IAM auth to authenticate against the vault server and expects the vault auth plugin to be mounted on /aws. Vault client locally will expect VAULT_TOKEN to be set.
+
+To configure your vault server to accept IAM Auth authentication, configure vault to use aws auth following https://learn.hashicorp.com/vault/identity-access-management/iam-authentication
+
+To configure kops to use the Vault store, add this to the spec:
+
+```yaml
+spec:
+  secretStore: vault://<vault>:<port>/secret/clusters/<clustername>/secrets
+  keyStore: vault://<vault>:<port>/secret/clusters/<clustername>/keys
+```
```

Comment:
Perhaps make it clear that the path to the secrets/keys is an example, not a requirement?

Reply:
Tried to make this more clear.